pom.xml

```diff
@@ -17,10 +17,151 @@
     <name>Spring Boot - Cache Booster</name>
     <description>Spring Boot - Cache Booster</description>
 
+  <properties>
+    <arquillian.version>1.4.0.Final</arquillian.version>
+    <junit.version>4.12</junit.version>
+    <spring-boot-bom.version>1.5.13.Final</spring-boot-bom.version>
+    <spring-boot.version>1.5.13.RELEASE</spring-boot.version>
+  </properties>
+
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>A business-friendly OSS license</comments>
+    </license>
+  </licenses>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>me.snowdrop</groupId>
+        <artifactId>spring-boot-bom</artifactId>
+        <version>${spring-boot-bom.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.arquillian</groupId>
+        <artifactId>arquillian-bom</artifactId>
+        <version>${arquillian.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>${junit.version}</version>
+        <scope>import</scope>
```

Comment:
This is wrong: scope should be `test` but I see you've fixed that in the scripts.

Reply:
Yup.

I updated the scripts and I am soon going to commit the updates to the boosters